docs/source/python_api/index.rst

```diff
@@ -3,13 +3,14 @@
 Python API
 ==========
 
-The MLflow Python API is organized into the following modules. The most common functions are also
+The MLflow Python API is organized into the following modules. The most common functions are
 exposed in the :py:mod:`mlflow` module, so we recommend starting there.
 
 .. toctree::
-    :glob:
+  :glob:
+  :maxdepth: 1
```

Comment:
What's the rationale here for restricting the depth? The current max depth of the doc tree is 2, which doesn't seem excessive.

Reply:
Because then subsections of mlflow.pyfunc display, but none of the other modules have subsections and it looks weird. https://www.mlflow.org/docs/latest/python_api/index.html

Reply:
I would rather prioritize making this content visible to users over creating depth consistency with other modules that do not happen to have subsections. Can we remove `maxdepth: 1`?